src/integrationTest/groovy/wooga/gradle/wdk/unity/PublishWdkUnityIntegrationSpec.groovy

```diff
@@ -89,7 +89,7 @@ class PublishWdkUnityIntegrationSpec extends com.wooga.gradle.test.IntegrationSp
         assert packageDir.exists()
 
         and: "configuration of the main plugin and the artifactory plugin"
-        String nugetKey = System.getenv("NUGET_KEY")
+        String nugetKey = System.getenv("repositoryCredentials") ?: System.getenv("NUGET_KEY")
```

Comment:
We should not use NUGET_KEY at all. Or bind it to a scoped credential in the Jenkinsfile / locally.

Reply:
Alright. I had set it this way for local testing.